client/src/Curious.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './App.css';
+import SVG from './SVG/svg.js'
+import Header from './Header'
+import About from './About'
```

Comment:
These import statements can be removed from here since SVG, Header, and About are not being used in this component.